metrics/firebase/test/firestore/rules/project-groups-test.js

```diff
@@ -113,7 +316,7 @@ describe("", async function () {
     {
       'describe': 'Authenticated with google and not allowed email domain user with not verified email',
       'app': await getApplicationWith(
-        getAllowedEmailUser(googleSignInProviderId, false)
+          getAllowedEmailUser(googleSignInProviderId, false)
```

Comment:
Please make sure to not check in unnecessary formatting.
```suggestion
        getAllowedEmailUser(googleSignInProviderId, false)
```